### PR TITLE
Coverity version bump & triage

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: ci/Dockerfile.coverity [hunk 1]
```diff
@@ -41,10 +41,10 @@ ARG ENV_COVERITY_PROJECT
...
 ARG ENV_BUILD_VERSION=3
 WORKDIR /
-RUN curl --form token=$ENV_COVERITY_TOKEN \
-  --form email=dariusz.trawinski@intel.com \
-  --form file=@model_server.tgz \
-  --form version=$ENV_BUILD_VERSION \
-  --form description="SDL build" \
-  https://scan.coverity.com/builds?project=$ENV_COVERITY_PROJECT
-
+#RUN curl --form token=$ENV_COVERITY_TOKEN \
```
Comment (atobiszei) [inline ci/Dockerfile.coverity:44]: ?

**Comment by atobiszei** [comment]
I think this can be pushed to next release

File: ci/Dockerfile.coverity (hunk 1 above)
Comment (dkalinowski) [reply ci/Dockerfile.coverity]: fixed

File: ci/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+cov-analysis-linux64*
+model_server.tar.gz
+response
```
[the remark this answers is not in the document]
Comment (dkalinowski) [reply ci/.gitignore:3]: This gitignore is convenience for us when running coverity scans manually.
- `model_server.tar.gz` is scan result (usually ~4gb) - we do not want to commit it to repo
- `response` is coverity artifact when uploading the scan to the cloud

**Review by dtrawins — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

**Pull request merged**